### DISCUSSION TITLE
Godot AnimationPlayer error

### DISCUSSION THREAD

**Discussion by GINZUcz** [godotengine/godot-proposals#8429] [GUI]
Hello! I got little problem here and i do not know how to solve it. Can you help me please?
![Snímek obrazovky 2023-11-14 153052](https://github.com/godotengine/godot-proposals/assets/84266450/32dd1679-2ba5-460a-bcb4-33fdb15ec7de)

**Comment by GINZUcz** [comment]
Forgot to say it started after adding 2 animation players and from here it started

**Comment by AThousandShips** [comment]
For support questions and help please first turn to the other [community channels](https://godotengine.org/community), this is for featur discussion, not a help desk. Please use other channels instead of here.

Bug reports are done [here](https://github.com/godotengine/godot/issues)

**Reply by GINZUcz** [reply]
oops! Sorry my bad! and thanks
